### PR TITLE
Agent configuration options App Analytics tracing

### PR DESCRIPTION
**Pull request by dussault-antoine**
### What does this PR do?

Document Agent configuration options App Analytics tracing

### Motivation

Missing Documentation for Agent configuration options App Analytics tracing

### Preview

https://docs-staging.datadoghq.com/antoine.dussault/app_analytics_agent/tracing/legacy_app_analytics

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/tracing/legacy_app_analytics/_index.md [hunk 1]
```diff
@@ -19,12 +19,11 @@ To migrate to the new configuration options, remove all legacy App Analytics [co
 
 ## App Analytics setup
 
-To enable App Analytics, either [automatically](#automatic-configuration) or [manually](#custom-instrumentation), configure your APM tracer to emit the relevant analytics from your services.
+App Analytics configuration options are located in the tracing libraries and in the Datadog Agent. In the libraries, analytics spans from your services are generated either [automatically](#automatic-configuration) or [manually](#custom-instrumentation).
 
+### In Tracing Libraries
```
Comment (buraizu) [inline content/en/tracing/legacy_app_analytics/_index.md:24]:
```suggestion
### In tracing libraries
```

Comment (buraizu) [reply content/en/tracing/legacy_app_analytics/_index.md:24]: Not sure if 'tracing libraries' is meant to be a proper noun, could you confirm?

Comment (dussault-antoine) [reply content/en/tracing/legacy_app_analytics/_index.md:24]: Let's keep the capital case, we do that in other places :)

File: content/en/tracing/legacy_app_analytics/_index.md [hunk 2]
```diff
@@ -19,12 +19,11 @@ To migrate to the new configuration options, remove all legacy App Analytics [co
 
 ## App Analytics setup
 
-To enable App Analytics, either [automatically](#automatic-configuration) or [manually](#custom-instrumentation), configure your APM tracer to emit the relevant analytics from your services.
+App Analytics configuration options are located in the tracing libraries and in the Datadog Agent. In the libraries, analytics spans from your services are generated either [automatically](#automatic-configuration) or [manually](#custom-instrumentation).
```
Comment (buraizu) [inline content/en/tracing/legacy_app_analytics/_index.md:22]:
```suggestion
App Analytics configuration options are located in the Tracing Libraries and in the Datadog Agent. In the libraries, analytics spans from your services are generated either [automatically](#automatic-configuration) or [manually](#custom-instrumentation).
```

Comment (buraizu) [reply content/en/tracing/legacy_app_analytics/_index.md:22]: For consistency

**Review by buraizu — APPROVED** [review]
Approval from docs team

**Comment by dussault-antoine** [comment]
We can merge 👍 thanks!

**Pull request merged**